### PR TITLE
Add link checker and fix broken links in getting started

### PR DESCRIPTION
**Pull request by oleksis**
- Added the [lychee-action](https://github.com/lycheeverse/lychee-action) GitHub Action to check for broken links in Markdown

Can check the [checklinks branch](https://github.com/oleksis/matic-docs/blob/checklinks/.github/workflows/links-fail-fast.yml) for check all the links in Markdown files

### CODE REVIEW

**Comment by oleksis** [comment]
[links_fail_fast.yml](https://github.com/oleksis/matic-docs/blob/linkchecker-py/.github/workflows/links_fail_fast.yml) version using LinkChecker for Pyhon

File: .github/workflows/links-fail-fast.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1
+        with:
+          # Check all markdown in docs/develop
+          args: --verbose --no-progress './docs/develop/getting-started.md'
```
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:17]: `args: --verbose --no-progress './docs/'`?

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thank you for the contribution @oleksis

File: .github/workflows/links-fail-fast.yml (hunk 1 above)
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:17]:
```suggestion
          args: --verbose --no-progress './docs/
```

**Review by DannyS03 — APPROVED** [review]

File: .github/workflows/links-fail-fast.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1
+        with:
+          # Check all markdown in docs/develop
+          args: --verbose --no-progress './docs/
```
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:17]:
```suggestion
          args: --verbose --no-progress './docs
```

Comment (oleksis) [reply .github/workflows/links-fail-fast.yml]: the args in docs dir need 2 apostrofe './docs'

File: .github/workflows/links-fail-fast.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1
+        with:
+          # Check all markdown in docs/develop
+          args: --verbose --no-progress './docs
```
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:17]:
```suggestion
          args: --verbose --no-progress './docs/'
```

**Comment by oleksis** [comment]
If [some files](https://github.com/maticnetwork/matic-docs/runs/8185517033?check_suite_focus=true#step:3:1904) builded in the build workflow the Link workflow will run after

File: .github/workflows/links-fail-fast.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+name: Links (Fail Fast)
+
+on:
+  push:
+  pull_request:
```
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:5]:
```suggestion
```

Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:5]:
```suggestion
  workflow_dispatch:
```

File: .github/workflows/links-fail-fast.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+name: Links (Fail Fast)
+
+on:
+  push:
```
Comment (DannyS03) [inline .github/workflows/links-fail-fast.yml:4]:
```suggestion
```

**Pull request merged**